src/main/java/uk/gov/companieshouse/web/accounts/controller/smallfull/StepsToCompleteController.java

```diff
@@ -0,0 +1,37 @@
+package uk.gov.companieshouse.web.accounts.controller.smallfull;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import uk.gov.companieshouse.api.model.transaction.Transaction;
+import uk.gov.companieshouse.web.accounts.controller.BaseController;
+import uk.gov.companieshouse.web.accounts.service.transaction.TransactionService;
+import uk.gov.companieshouse.web.accounts.util.RequestPathResolver;
+
+@Controller
+@RequestMapping(value = "/company/{companyNumber}/small-full/steps-to-complete")
+public class StepsToCompleteController {
+
+    private static final String TEMPLATE = "smallfull/stepsToComplete";
+
+    @Autowired
+    TransactionService transactionService;
+
+    @GetMapping()
+    public String getStepsToComplete() {
+
+        return TEMPLATE;
+    }
+
+    @PostMapping()
+    public String postStepsToComplete(@PathVariable String companyNumber) {
+
+        Transaction transaction = transactionService.createTransaction(companyNumber);
+
+        // TODO: perform a redirect when a navigation engine is implemented
```

Comment:
Sorry to be picky: can you remove the "engine" from the wording of the TODO please? I would prefer to keep it simple and avoid needing an engine.

Reply:
Done